deepchem/models/tests/test_layers.py

```diff
@@ -1262,3 +1263,55 @@ def test_mxmnet_besselbasis():
     result = np.array([[2.6434e+00, 3.7383e+00], [1.3125e+00, -1.1474e-07],
                        [-0.0000e+00, 0.0000e+00], [-0.0000e+00, -0.0000e+00]])
     assert np.allclose(result, output, atol=1e-04)
+
+
+@pytest.mark.torch
+def test_mxmnet_local_message_passing():
+    """ Test for MXMNetGlobalMessagePassing Layer."""
+    seed = 123
+    torch.backends.cudnn.deterministic = True
+    torch.backends.cudnn.benchmark = False
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+
+    dim = 1
+    h = torch.tensor([[0.8343], [1.2713], [1.2713], [1.2713], [1.2713]])
+
+    rbf = torch.tensor([[-0.2628], [-0.2628], [-0.2628], [-0.2628], [-0.2629],
+                        [-0.2629], [-0.2628], [-0.2628]])
+
+    sbf1 = torch.tensor([[-0.2767], [-0.2767], [-0.2767], [-0.2767], [-0.2767],
+                         [-0.2767], [-0.2767], [-0.2767], [-0.2767], [-0.2767],
+                         [-0.2767], [-0.2767]])
+
+    sbf2 = torch.tensor([[-0.0301], [-0.0301], [-0.1483], [-0.1486], [-0.1484],
+                         [-0.0301], [-0.1483], [-0.0301], [-0.1485], [-0.1483],
+                         [-0.0301], [-0.1486], [-0.1485], [-0.0301], [-0.1486],
+                         [-0.0301], [-0.1484], [-0.1483], [-0.1486], [-0.0301]])
+
+    idx_kj = torch.tensor([3, 5, 7, 1, 5, 7, 1, 3, 7, 1, 3, 5])
+
+    idx_ji_1 = torch.tensor([0, 0, 0, 2, 2, 2, 4, 4, 4, 6, 6, 6])
+
+    idx_jj = torch.tensor(
+        [0, 1, 3, 5, 7, 2, 1, 3, 5, 7, 4, 1, 3, 5, 7, 6, 1, 3, 5, 7])
+
+    idx_ji_2 = torch.tensor(
+        [0, 1, 1, 1, 1, 2, 3, 3, 3, 3, 4, 5, 5, 5, 5, 6, 7, 7, 7, 7])
+
+    edge_index = torch.tensor([[0, 1, 0, 2, 0, 3, 0, 4],
+                               [1, 0, 2, 0, 3, 0, 4, 0]])
+
+    out = dc.models.torch_models.layers.MXMNetLocalMessagePassing(
```

Comment:
name this variable as 'layer' instead of 'out; 'out' makes it confusing to interpret

Reply:
Okay sure I'll do it.